packages/bindings/src/query.rs

```diff
@@ -5,7 +5,7 @@ use serde::{Deserialize, Serialize};
 use crate::types::ProvenanceRoute;
 
 /// Represents a request to query a custom provenance module.
-#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[derive(Serialize, Deserialize, Clone, Debug, Eq, PartialEq, JsonSchema)]
```

Comment:
Same comments as attrs/src/msg.rs.

Reply:
the bindings package actually remains unchanged. only inputs/outputs of the contracts use the new derived schema